rhizome/postgres/lib/postgres_setup.rb

```diff
@@ -17,6 +17,29 @@ def install_packages
     end
   end
 
+  def configure_swap
```

Comment:
cloudinit has swap options. You don't setup manually. We use it for runners

Reply:
`Prog::Vm::Nexus.assemble` has `swap_size_bytes` param
https://github.com/ubicloud/ubicloud/blob/7ef3e6abdf10c26253f1972ee64ca26d3a15dd9d/prog/vm/nexus.rb#L17